ethcore/builtin/Cargo.toml

```diff
@@ -17,6 +17,7 @@ log = "0.4"
 num = { version = "0.1", default-features = false, features = ["bigint"] }
 parity-bytes = "0.1"
 parity-crypto = { version = "0.6.1", features = ["publickey"] }
+eth_pairings = { git = "https://github.com/matter-labs/eip1962.git", default-features = false, features = ["eip_2537"] }
```

Comment:
If not published in crates, will be nice to specify the `rev` (`1c79dba99c970c99066e249627c29b38dc949677` ?) or the `tag` used. It will be the crate version to be checked, also.